static/js/components/EducationTab.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Button from 'react-mdl/lib/Button';
+import ProfileTab from "../util/ProfileTab";
+import { boundEducationSelectField,
+  boundEducationMonthField, boundEducationYearField,
+  saveAndContinue } from '../util/profile_edit';
+import {
+  toggleEducationLevel,
+  openEducationForm,
+  closeEducationForm
+} from '../actions';
+import { Grid, Cell, Switch, Dialog, DialogTitle, DialogContent, DialogActions, FABButton, Icon } from 'react-mdl';
+
+class EducationTab extends ProfileTab {
+  constructor(props) {
+    super(props);
+    this.boundEducationMonthField = boundEducationMonthField.bind(this);
+    this.boundEducationYearField = boundEducationYearField.bind(this);
+    this.educationLevelOptions = [
+      { value: 'hs', label: "High school", enabled: true },
+      { value: 'a', label: 'Associate degree', enabled: true },
+      { value: 'b', label: "Bachelor's degree", enabled: true },
+      { value: 'm', label: "Master's or professional degree", enabled: false },
+      { value: 'p', label: "Doctorate", enabled: false }
+    ];
+    this.educationLevelLabels = {
+      'hs' : "High school",
+      'a': 'Associate degree',
+      'b': "Bachelor's degree",
+      'm': "Master's or professional degree",
+      'p': "Doctorate"
+    };
+
+    this.saveAndContinue = saveAndContinue.bind(this, '/dashboard');
+    this.handleOpenDialog = this.handleOpenDialog.bind(this);
+    this.handleCloseDialog = this.handleCloseDialog.bind(this);
+  }
+  printAddDegree(level){
+    const { educationLevels } = this.props;
+    if (educationLevels[level.value]){
+      return <Grid key={"add-"+level.value}>
+        <Cell col={11}></Cell>
+        <Cell col={1}>
+          <FABButton mini onClick={this.handleOpenDialog.bind(this, level.value, null )} raised ripple>
+            <Icon name="add" />
+          </FABButton>
+        </Cell>
+
+      </Grid>;
+    }
+  }
+  printExistingEducations(level){
+    const { profile } = this.props;
+    return profile['education'].map(education => {
+      if(education.degree_name === level.value && 'id' in education) {
```

Comment:
I like the `'id' in education` here. We should remove the `id: null` default from the employment tab default data so the two are consistent